_docs/examples/ios.md

```diff
@@ -101,6 +101,8 @@ Explanation [here](https://gist.github.com/dpnishant/c7c6b47ebfd8cd671ecf).
 
 ## Data Structures
 
+>**Tip**: If things don't seem to be working as expected you may be interacting with the wrong data type - run `log("Type of args[2] -> " + ObjC.Object(args[2]).$className)` to check the type of class that was decoded!
```

Comment:
- `log()` -> `console.log()` so this example also works outside of frida-trace.
- `ObjC.Object` should always be instantiated with `new`.
- Perhaps this is clearer: "to check the type of object that you're dealing with"

Reply:
Makes sense! Changes incoming.